Userland/Utilities/hexdump.cpp

```diff
@@ -59,20 +67,49 @@ int main(int argc, char** argv)
     };
 
     Array<u8, BUFSIZ> contents;
+    Span<u8> prev_line {};
```

Comment:
This has a default constructor, no need to explicitly initialise it like this.
Also, please use full words for variable names (`previous_line`)